safehttp/response_writer.go

```diff
@@ -122,5 +169,7 @@ func (w *ResponseWriter) SetCookie(c *Cookie) error {
 // Dispatcher TODO
 type Dispatcher interface {
 	Write(rw http.ResponseWriter, resp Response) error
+	WriteJSON(rw http.ResponseWriter, resp JSONResponse) error
 	ExecuteTemplate(rw http.ResponseWriter, t Template, data interface{}) error
+	ContentType(resp Response) (string, error)
```

Comment:
Start adding comments to the methods. You don't have to add to the ones that existed before your PR, we can do that later.